internal/api/mfa_test.go

```diff
@@ -384,7 +384,7 @@ func (ts *MFATestSuite) TestUnenrollUnverifiedFactor() {
 func (ts *MFATestSuite) TestSessionsMaintainAALOnRefresh() {
 	email := "test1@example.com"
 	password := "test123"
-	token := signUpAndVerify(ts, email, password)
+	token := signUpAndVerify(ts, email, password, true /* <- isSuccessGuaranteed */)
```

Comment:
```suggestion
	token := signUpAndVerify(ts, email, password, true /* <- requireStatusOK */)
```


Maybe change to this across the board